client/components/home/UserInput/Items.tsx

```diff
@@ -30,14 +30,20 @@ export const Items = ({ items, handleUpdateItem, handleAddItem, handleRemoveItem
 			<VStack spacing="10px">
 				{items.map((item, index) => (
 					<HStack key={`item-${index}`} spacing="6px" width="100%">
-						<Input
+						<StyledSelect
 							placeholder="Item"
-							_placeholder={{ color: COLORS.Input.placeholder }}
+							color={item.name ? "chakra-body-text" : COLORS.Input.placeholder}
+							iconColor={COLORS.Select.icon}
 							value={item.name}
-							onChange={(e) => {
+							onChange={(e: ChangeEvent<HTMLInputElement>) => {
```

Comment:
this one actually don't need to type, because it should be inferred from StyledSelect prop tyles

Reply:
Okay ill edit it back

Reply:
sorry, i realised that for some reason the event is not implicitly typed, most likely a chakra problem, and also that i previously typed the other StyledSelect.

so, i've reverted the change to any. anyway, not good to type it as any.